File: java/Ejercicio41.java
import java.util.Scanner;


public class Ejercicio41 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        

        System.out.print("Ingrese el volumen del gas (L): ");
        double volumen = sc.nextDouble();

          System.out.print("Ingrese la temperatura del gas (K): ");
        double temperatura = sc.nextDouble();

        System.out.print("Ingrese la cantidad de sustancias en moles  (mol): ");
        double n = sc.nextDouble();

        final double r= 0.0821;
        
        double presion = n * r * temperatura/volumen;  

        System.out.println("La presion es: " + presion + " atm");

        sc.close();
    }
}

File: python/Ejercicio41.py
volumen = float(input("Ingrese el volumen del gas (L): "))
n = float(input("Ingrese la cantidad de sustancia en moles (mol): "))
temperatura = float(input("Ingrese la temperatura (K): "))
r = 0.0821 

presion = n * r * temperatura / volumen

print ("La presion del gas  es :", presion, "atm" )